### PR TITLE
Update win10-vm.md

### PR DESCRIPTION
**Pull request by asklar**
Add Azure VM option

### CODE REVIEW

File: docs/win10-vm.md [hunk 1]
```diff
@@ -3,11 +3,18 @@ id: win10-vm
 title: Developing Windows apps on a non-Windows PC
 ---
 
-It is possible to develop Windows apps on a non-Windows PC by creating using a virtual machine.
+It is possible to develop Windows apps on a non-Windows PC by creating using a virtual machine either locally or on Azure.
```
Comment (rectified95) [inline docs/win10-vm.md:6]: >by creating using a virtual machine

Probably wanted to remove part of that

File: docs/win10-vm.md [hunk 2]
```diff
@@ -3,11 +3,18 @@ id: win10-vm
...
 
 ![](assets/powershell-start-menu.png)
 
-3. At this point you are ready to set up your development dependencies. Follow the instructions on [dependencies](rnw-dependencies.md).
\ No newline at end of file
+3. At this point you are ready to set up your development dependencies. Follow the instructions on [dependencies](rnw-dependencies.md).
+
+### Azure VM
+
+1. Go to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/using-visual-studio-vm and follow the instructions to create a Visual Studio 2019 on Windows 10 in Azure.
```
Comment (rectified95) [inline docs/win10-vm.md:19]: >follow the instructions to create a Visual Studio 2019 on Windows

missing word 'vm'? or 'vm with vs installed'?

**Review by rectified95 — APPROVED** [review]
Nice addition. Go over the missing words :)

File: docs/win10-vm.md (hunk 1 above)
Comment (asklar) [reply docs/win10-vm.md:6]:
```suggestion
It is possible to develop Windows apps on a non-Windows PC by using a virtual machine. The virtual machine can run either locally or on Azure.
```

File: docs/win10-vm.md (hunk 2 above)
Comment (asklar) [reply docs/win10-vm.md:19]:
```suggestion
1. Go to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/using-visual-studio-vm and follow the instructions to create a **Visual Studio 2019 Latest** virtual machine on "Windows 10 Enterprise N" in Azure.
```

**Pull request merged**